src/native/corehost/fxr/fx_resolver.h

```diff
@@ -97,6 +94,9 @@ class fx_resolver_t
 
     bool m_disable_multilevel_lookup;
     const runtime_config_t::settings_t& m_override_settings;
+
+    // Disabled runtime versions
+    std::vector<pal::string_t> m_disabled_versions;
```

Comment:
What is the lifetime of this? Is it for the lifetime of the process or is it freed after activation?